xml/System.IO/WindowsRuntimeStorageExtensions.xml

```diff
@@ -20,8 +20,92 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>
+      <format type="text/markdown"><![CDATA[
+      Contains extension methods for the <see xref="T:Windows.Storage.IStorageFile"/> and <see xref="T:Windows.Storage.IStorageFolder"/> interfaces in the Windows Runtime when developing Windows Store apps.
+      ]]></format>
+    </summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+      ## Remarks
+
+These extension methods are available only when you develop Windows Store apps. The methods provide convenient ways of opening files for reading or writing in Windows Store apps. You do not create an instance of the WindowsRuntimeStorageExtensions class; instead, you use these methods from instances of the IStorageFile and IStorageFolder interfaces.
+
+The WindowsRuntimeStorageExtensions class contains two methods that extend IStorageFile for reading or writing:
+
+  - [OpenStreamForReadAsync(IStorageFile)](/dotnet/api/system.io.windowsruntimestorageextensions.openstreamforreadasync?view=dotnet-uwp-10.0#System_IO_WindowsRuntimeStorageExtensions_OpenStreamForReadAsync_Windows_Storage_IStorageFile_)
+  - [OpenStreamForWriteAsync(IStorageFile)](/dotnet/api/system.io.windowsruntimestorageextensions.openstreamforreadasync?view=dotnet-uwp-10.0#System_IO_WindowsRuntimeStorageExtensions_OpenStreamForReadAsync_Windows_Storage_IStorageFolder_System_String_)
+
+The WindowsRuntimeStorageExtensions class contains two methods that extend IStorageFolder for reading and writing:
+  - [OpenStreamForReadAsync(IStorageFolder, String)](/dotnet/api/system.io.windowsruntimestorageextensions.openstreamforwriteasync?view=dotnet-uwp-10.0#System_IO_WindowsRuntimeStorageExtensions_OpenStreamForWriteAsync_Windows_Storage_IStorageFile_)
+  - [OpenStreamForWriteAsync(IStorageFolder, String, CreationCollisionOption)](/dotnet/api/system.io.windowsruntimestorageextensions.openstreamforwriteasync?view=dotnet-uwp-10.0#System_IO_WindowsRuntimeStorageExtensions_OpenStreamForWriteAsync_Windows_Storage_IStorageFolder_System_String_Windows_Storage_CreationCollisionOption_)
+
+## Examples
+
+The following example shows how to open a file in the application data as a [Stream](/dotnet/api/system.io.stream) in a Windows Store app, and write to it by using an instance of the [StreamWriter](/dotnet/api/system.io.streamwriter) class. It then reads the contents for the file by using an instance of the [StreamReader](/dotnet/api/system.io.streamreader) class.
+
+```C#
```

Comment:
Let's switch the inline code to references to the existing samples in dotnet/samples repo. For example:

```markdown
[!code-csharp[windowsruntimestorageextensions#8](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.io.windowsruntimestorageextensions/cs/blankpage4.xaml.cs#8)]
```

I thought there would be the VB version there too but I couldn't find it right now.

Let me know if you need help searching for the right ones on that repo.

Reply:
These are all in https://github.com/dotnet/samples?

Reply:
They should be! Search for part of the code and see if you can find it.